build.gradle

```diff
@@ -336,7 +275,7 @@ dependencies {
     // Findbugs annotations, so we can selectively suppress findbugs findings
     compileOnly 'com.google.code.findbugs:annotations:3.0.1'
```

Comment:
Can this be removed now? And I see another reference on line 282 below.

Reply:
Removed.